programmers/난이도별/level01.모의고사/yoojinhee03.java

```diff
@@ -0,0 +1,48 @@
+package programmers;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class yoojinhee03 {
+
+	public static void main(String[] args) {
+		int[] arr= {1,2,3,4,5};
+		for(int i:solution(arr)) {
+			System.out.println(i);
+		}
+	}
+	
+	 public static int[] solution(int[] answers) {
+        int[] answer = {};
+        int stu[][]= 
+        	{
+        			{1,2,3,4,5},//5
+        			{2,1,2,3,2,4,2,5},
+        			{3,3,1,1,2,2,4,4,5,5,},
+        	};
+        int[] count = new int[stu.length];
+        
+        for(int i=0,ansLen=answers.length; i<ansLen; i++) {
+        	if(stu[0][i%stu[0].length]==answers[i]) count[0]++;
+        	if(stu[1][i%stu[1].length]==answers[i]) count[1]++;
+        	if(stu[2][i%stu[2].length]==answers[i]) count[2]++;

```

Comment:
```suggestion
        	if(stu[0][i%5]==answers[i]) count[0]++;
        	if(stu[1][i%8]==answers[i]) count[1]++;
        	if(stu[2][i%10]==answers[i]) count[2]++;
```
어차피 고정된 값이기 때문에 이렇게 5, 8, 10 으로만 해도 좋을 것 같아요~